### PR TITLE
Merge in new verification and archive JJobs for rocoto workflow

### LINKED ISSUE #720: Creating J jobs for archive
**Description**

Create JJOBs for archive
**Requirements**

The current rocoto workflow run archive and verification jobs using scripts under $HOMEgfs/jobs/rocoto. Manager has requested to create JJOBs for archive and verification jobs. Modify the $HOMEgfs/jobs/rocoto scripts to call the new JJOBs.
**Acceptance Criteria (Definition of Done)**

For workflow to run archive and verification jobs using JJOBs.

Verify jobs will be handled in the issue #235

### PR DESCRIPTION
**Pull request by lgannoaa**
**Description**
jobs/JGFS_ATMOS_CYCLONE_TRACKER - TRACKER change need for new version
jobs/JENKFGDAS_ARCHIVE - A new JJob for earc (Tested with rocoto workflow)
jobs/JGLOBAL_ARCHIVE - A new JJob for GDAS/GFS archive (Tested with rocoto workflow)
jobs/JGLOBAL_VRFY - A new JJob for GDAS/GFS verification (Tested with rocoto workflow)
parm/config/config.vrfy - verification configuration file for rocoto workflow GDAS/GFS vrfy jobs
jobs/rocoto/arch.sh - rocoto workflow archive job script changed to call JJob (Tested with rocoto workflow)
jobs/rocoto/earc.sh - rocoto workflow earc job script changed to call JJob (Tested with rocoto workflow)
jobs/rocoto/vrfy.sh - rocoto workflow verification job script changed to call JJob (Tested with rocoto workflow)
These changes and new jobs have been tested with rocoto workflow.

closes #720 , #725 

**Type of change**

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

**How Has This Been Tested?**

  
**Checklist**

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [X] My changes generate no new warnings

### CODE REVIEW

File: parm/config/config.vrfy [hunk 1]
```diff
@@ -101,7 +104,7 @@ fi
 
 if [ $VRFYRAD = "YES" -o $VRFYMINMON = "YES" -o $VRFYOZN = "YES" ]; then
 
-    export envir="para"
+    export envir="prod"
```
Comment (WalterKolczynski-NOAA) [inline parm/config/config.vrfy]: Why the change?

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]
The new j-jobs are too involved. j-jobs aren't supposed to really do much other than setting variables. Most of the code needs to be pushed down a level into a new script-level script. This could theoretically be done in a follow-up PR, but I'd rather have it done correctly from the beginning.

File: parm/config/config.vrfy (hunk 1 above)
Comment (lgannoaa) [reply parm/config/config.vrfy]: The radmon ,minmon, and oznmom used internal variable envir that need to be set on this level.

Comment (WalterKolczynski-NOAA) [reply parm/config/config.vrfy]: That's why it is set here, but why was it _changed_

Comment (lgannoaa) [reply parm/config/config.vrfy]: I will drop this change. Do you have any other question on this PR before I update it?

Comment (WalterKolczynski-NOAA) [reply parm/config/config.vrfy]: Just the general comment about the j-jobs, but I didn't really go through those carefully, though I assume they are just what was previously in the rocoto/<task> script.

Comment (lgannoaa) [reply parm/config/config.vrfy]: Dropped

**Comment by WalterKolczynski-NOAA** [comment]
Has there been any progress in sliming the j-job down and moving code that doesn't do stuff into lower-level scripts? This PR has been sitting idle for a while.

**Comment by lgannoaa** [comment]
@WalterKolczynski-NOAA We had group meeting before. It was discussed that this solution is requested by the management. There is no plan to change the requirement at this time.

**Comment by aerorahul** [comment]
Closing after discussions w/ @lgannoaa
